nemoguardrails/library/domain_hallucination/layer2_advanced.py

```
@@ -0,0 +1,172 @@
+            prompt,
+            stop=stop,
+            llm_params={
+                "temperature": config.lowest_temperature,
+                "max_tokens": max_tokens,
+            },
+        )
+
+        warn_if_truncated(llm_response, task_name)
+        response_text = llm_response.content.strip().lower()
+
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)